index.html

```diff
@@ -1,13 +1,83 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
-    <meta charset="utf-8" />
-
-    <link href="styles.css" rel="stylesheet" />
-
-    <title>Rocket Academy</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Kent's Portfolio Page</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
   </head>
   <body>
-    <h1>Hello, world!</h1>
+    <nav>
+      <div class="navbar">
+        <a href="#about" class="icon"><img src="./images/avataaars.png"></a>
+        <ul class="categories">
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="./projects.html">Projects</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <div class="about" id="about">
+      <h1 class="header">About Me</h1>
+      <div class="pic">
+        <img src="./images/avataaars.png">
+        <p class="name">Kent Loo</p>
+        <p class="sch">Year 2 Computer Science Student at SIM UOL</p>
+        <p class="country">Based in Singapore</p>
+      </div>
+
+      <article class="intro">
+        <p>Hi, I'm Kent Loo, a second-year Computer Science Student at SIM UOL, based in Singapore. I have a passion for software development and enjoy learning how to code and create applications.</p>
+        <p>Currently, I'm pursuing a BSc in Computer Science and also attending a bootcamp at Rocket Academy, where i'm honing my skills in data structures, algorithms and full-stack web development.</p>
+        <p>I've worked on a few projects using languages like JavaScript, CSS, HTML, C++ and Python. Through these projects, I hope to develop skills in both front-end and back-end technologies, and I'm eager to continue learning and improving.</p>
+        <p>When I'm not coding, I enjoy playing sports, gaming and watching anime.</p>
+        <p>Feel free to explore my work and get in touch if you'd like to discuss opportunities or collaborate on exciting projects!</p>
+      </article>
+    </div>
+
+    <div class="recent">
+      <h1 class="header">Recent Projects</h1>
+      <div class="repo">
+        <div class="project">
+          <a href="https://kentkentucky.github.io/Snooker-Game/"><img src="./images/snooker.jpg">Snooker Game</a>
```

Comment:
The game doesn't work? :(

Reply:
the game works. 
click 1,2,3 or 4 for different game modes.
to spawn the cue ball. mouse has to be in D and click key W.
to shoot, use the space bar.